src/word-search/index.js

```diff
@@ -1,3 +1,59 @@
-function wordSearch(words, word) {}
+function wordSearch(words, word) 
```

Comment:
fix your failing test cases

Reply:
Sir, I have updated my branch with the new test cases.